utils/check_api_reference.py

```diff
@@ -121,6 +121,72 @@
     print(extra)  # noqa: T201
     ret = 1
 
+# Series.cat methods
```

Comment:
is it possible to loop over `['cat', 'dt', 'str', 'name']` for these?

Reply:
Sure! Btw, a `.md` for `Series.name` is missing at all? Is this expected?

Reply:
Sure! If I'm not wrong, `"name"` isn't a real namespace for `Series` as it is for `Expr`, so I excluded it from the loop. lmk if that's ok